### PR TITLE
Add total classification statistic

### LINKED ISSUE #38: Add user statistic - Total Classifications
Add a user statistic below the user avatar of total # of classifications, e.g. "xx Total Classifications"

### PR DESCRIPTION
**Pull request by robinschaaf**
Fixes #38 

  *  Add total classification to ProjectDisciplines (home screen)
  *  Update test

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/components/ProjectDisciplines.js [hunk 1]
```diff
@@ -65,12 +65,19 @@ export class ProjectDisciplines extends React.Component {
...
 
+    const totalClassifications =
+      <StyledText
+        additionalStyles={[styles.totalClassifications]}
+        text={`${this.props.user.totalClassifications} total classifications`} />
+
     return (
       <View style={styles.container}>
         <View style={styles.subNavContainer}>
-          <Text style={styles.userName}>
```
Comment (wgranger) [inline src/components/ProjectDisciplines.js]: Can you remove this `Text` component from the RN imports now? I don't believe this component is using it any longer.

File: src/components/ProjectDisciplines.js [hunk 2]
```diff
@@ -65,12 +65,19 @@ export class ProjectDisciplines extends React.Component {
           text={'You must have an internet connection to use Zooniverse Mobile'} />
       </View>
 
+    const totalClassifications =
+      <StyledText
+        additionalStyles={[styles.totalClassifications]}
+        text={`${this.props.user.totalClassifications} total classifications`} />
```
Comment (wgranger) [inline src/components/ProjectDisciplines.js]: This is me being a grammar snob, but what happens when there is one total classification? Right now it's always total classifications (plural).

Comment (robinschaaf) [inline src/components/ProjectDisciplines.js]: Lol good point!

File: src/components/ProjectDisciplines.js [hunk 3]
```diff
@@ -65,19 +65,18 @@ export class ProjectDisciplines extends React.Component {
           text={'You must have an internet connection to use Zooniverse Mobile'} />
       </View>
 
-    const totalClassifications =
+    const pluralizeClassification = ( totalClassifications > 1 ? 's' : '' )
```
Comment (wgranger) [inline src/components/ProjectDisciplines.js]: this is a clever fix.

**Pull request merged**